apollo-runtime/src/main/java/com/apollographql/android/impl/RealApolloCall.java

```diff
@@ -110,10 +112,13 @@ private RealApolloCall(Operation operation, HttpUrl serverUrl, Call.Factory http
         httpCall.enqueue(new HttCallback(callback));
       }
     });
-    return this;
   }
 
-  @Nonnull @Override public ApolloCall<T> httpCacheControl(@Nonnull HttpCacheControl httpCacheControl) {
+  @Nonnull @Override public RealApolloWatcher<T> toWatcher() {
```

Comment:
With `clone`, this would be acceptable. I don't see a real harm in allowing it, but it does go against the explicit single-use immutable style of the other methods.